### PR TITLE
RNL - Set correct image width

### PR DESCRIPTION
**Pull request by Rohit-L**
Addresses last remaining task of issue #3 

1) Sets width of pictures on the images index page to be 400px

### CODE REVIEW

File: app/assets/stylesheets/images.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
 // Place all the styles related to the images controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.width-400-px {
+	width: 400px;
```
Comment (rahulrajaram) [inline app/assets/stylesheets/images.scss]: I am not sure if the indentation here is alright. [The raw format of this file](https://raw.githubusercontent.com/AppFolioOnboarding/Rohit/b3c5f0ecc451aa7cc47180be93ad742a3550c5fd/app/assets/stylesheets/images.scss) suggests that this line has a tab space.

If that is the case, verify in the future that your indentation settings are correct.

File: app/assets/stylesheets/images.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
 // Place all the styles related to the images controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.width-400-px {
```
Comment (rahulrajaram) [inline app/assets/stylesheets/images.scss]: This is not the best name, IMO, for what is actually the image. I would prefer that you rename the class to `image`.

Comment (Rohit-L) [inline app/assets/stylesheets/images.scss]: Okay, no problem. I fixed both things.

File: app/assets/stylesheets/images.scss [hunk 3]
```diff
@@ -1,3 +1,6 @@
 // Place all the styles related to the images controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.image {
+  width: 400px;
+}
```
Comment (rahulrajaram) [inline app/assets/stylesheets/images.scss]: It is a good idea to leave a newline at the end of each file so it is obvious that the file ends with a newline. You don't need to fix this.

**Pull request merged**